Source/monstdat.h

```diff
@@ -89,44 +89,44 @@ enum class MonsterAvailability : uint8_t {
 };
 
 struct MonsterData {
-	const char *mName;
-	const char *GraphicType;
-	const char *sndfile;
-	const char *TransFile;
+	const char *name;
+	const char *graphicType;
+	const char *sndFile;
+	const char *transFile;

```

Comment:
lets just call it `trnFile` since that is the file type, no need to have multiple acronyms for this.